saas/common/src/applatix.io/s3cl/s3cl.go

```diff
@@ -0,0 +1,89 @@
+package s3cl
+
+import (
+	"applatix.io/common"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3"
+	"net/http"
+	"os"
+)
+
+var s3Region, _ = os.LookupEnv("AX_REGION")
+var s3EndPoint, _ = os.LookupEnv("ARGO_S3_ENDPOINT")
```

Comment:
What happens if this env variable is not set? Is there a way to set some defaults?

Reply:
variables will be empty strings if env variables are not set. If s3Endpoint is empty i assume we are in aws and use the aws specific way of creating session.